ivy/functional/backends/tensorflow/experimental/layers.py

```diff
@@ -444,7 +444,33 @@ def avg_pool3d(
     return res
 
 
-@with_supported_dtypes({"2.12.0 and below": ("float32", "float64")}, backend_version)
+@with_unsupported_dtypes(
+    {"2.12.0 and below": ("bfloat16", "float64", "float16")}, backend_version
+)
```

Comment:
I think you have removed the `with_unsupported_dtypes` from the following function, `dct` and placed it for `pool`. Can you please fix that?